metaflow/plugins/argo/argo_workflows.py

```diff
@@ -143,6 +143,50 @@ def _sanitize(name):
         # allowed by Metaflow - guaranteeing uniqueness.
         return name.replace("_", "-")
 
+    @staticmethod
+    def remove_schedule(name):
```

Comment:
Can we move this inside the delete method in L:158 and move the comments about the behavior in cli.py to here?